config/bootstrap.php

```diff
@@ -26,4 +26,6 @@
 	require_once $filepath;
 }
 
+define('HIDE_EXAMPLES', empty($container->parameters['showExamples']));
```

Comment:
I don't think it's wise to introduce a `showExample` parameter but proceed to use the opposite as a constant ‒ it just adds unnecessary confusion.

Reply:
Good point. Changed constant to `SHOW_EXAMPLES`